Assets/MRTK/SDK/Features/Input/Handlers/ObjectManipulator.cs

```diff
@@ -419,6 +419,15 @@ private void Start()
             {
                 constraintsManager = gameObject.EnsureComponent<ConstraintManager>();
             }
+
+            // Get child objects with NearInteractionGrabbable attached
+            var children = GetComponentsInChildren<NearInteractionGrabbable>();
+
+            if (children.Length == 0)
+            {
+                Debug.LogWarning($"Near interactions are not enabled for {gameObject.name}. To enable near interactions, add a " +
+                    $"{nameof(NearInteractionGrabbable)} component to {gameObject.name} or to a child object of {gameObject.name} that contains a collider.");
+            }
```

Comment:
The current runtime warning will log any time a NearInteractionGrabbable is not present on the object or in its hierarchy.  Now it could be the case that the user intends to use object manipulator with only far interaction support, and this warning could be annoying.  

Although, after a discussion with @jbienzms it seems that the most likely use case for object manipulator would be for both near and far interaction support.

@julenka @RogPodge @keveleigh 
What are your thoughts?

Reply:
In the inspector, the warning is hidden unless the user opens the relevant field right? Is it possible to have the debug warning work in the same way, since I think its fair to say users are probably not be interested in near interaction unless they look there in the inspector.

Reply:
Yes the warning in the inspector is hidden if the user closes the foldout.  The same sort of behavior in a runtime warning would need a property to represent the indication of user intent to add support for near interactions. Currently, ObjectManipulator does not have a clear property to capture user intent for adding near interaction support.

Worst case, logging a warning in the console every time a NearInteractionGrabbable is not present could result in many warnings in existing projects that currently use Object Manipulator without a Near Interaction Grabbable.  But the assumption would be that Object Manipulator most commonly used with the intent of supporting both near and far interaction.

Reply:
I am thinking whether a boolean field could be added which controls whether to show this warning message or not. If the user knows what he is doing then he can just toggle this boolean to disable the warning in the inspector.

Reply:
The idea is basically the same as @CDiaz-MS mentioned above, though I was thinking about calling it something like `IgnoreNearInteractionWarning` but it could work either way. And if we do decide to have that field we probably should set it to show warnings by default.